Source/Scene/Cesium3DTileset.js

```diff
@@ -1468,8 +1527,23 @@ define([
      * @exception {DeveloperError} The tileset must be 3D Tiles version 0.0.  See https://github.com/AnalyticalGraphicsInc/3d-tiles#spec-status
      */
     Cesium3DTileset.prototype.update = function(frameState) {
+        if (!this._ready) {
```

Comment:
Is all this really needed?  Couldn't we just have `_ready` and move some logic into `frameState.afterRender` to preserve the logic of the first `update`?

Either way, there should be a comment so we don't try to remove the "needless" complexity later.